config/clock.rb

```diff
@@ -25,6 +25,10 @@ class Clock
   every(1.hour, 'SendChaseEmailToCandidates', at: '**:40') { SendChaseEmailToCandidatesWorker.perform_async }
   every(1.hour, 'UpdateFeatureMetricsDashboard', at: '**:45') { UpdateFeatureMetricsDashboard.perform_async }
 
+  # StartOfCycleNotificationWorker only works the day the service opens. These two jobs won't run at the same time on the same day.
+  every(1.hour, 'SendFindStartOfCycleProviderEmails', at: '**:50') { StartOfCycleNotificationWorker.perform_async(:find) }
```

Comment:
I was wondering if we could try`'**:05'` ?

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/5682/commits/d1ada6bd217d6bc09f8c1aa6a8ef722f56bcd2c0